graf2d/gpadv7/inc/ROOT/RAttrAxis.hxx

```diff
@@ -46,10 +46,11 @@ class RAttrAxis : public RAttrBase {
    RAttrValue<RPadLength> fTicksSize{this, "ticks_size", 0.02_normal};   ///<! ticks size
    RAttrValue<RColor> fTicksColor{this, "ticks_color", RColor::kBlack};  ///<! ticks color
    RAttrValue<int> fTicksWidth{this, "ticks_width", 1};                  ///<! ticks width
-   RAttrText fLabelsAttr{this, "labels"};                                ///<! text attributes for labels
+   RAttrValue<bool> fNoLabels{this, "nolabels", false};                  ///<! disable labels drawing
+   RAttrText fAttrLabels{this, "labels"};                                ///<! text attributes for labels
```

Comment:
What's the relationship between this and the label-axes, which have their own array of bin labels?

Reply:
This flag disable any kind of labels drawing

Reply:
Sorry, that one should have been obvious to me :-/